docker/edge/http/pep-config.js

```diff
@@ -0,0 +1,81 @@
+const config = {};
+
+function toBoolean(env, defaultValue) {
+  return env !== undefined ? env.toLowerCase() === 'true' : defaultValue;
+}
+
+function to_array(env, default_value){
+    return (env !== undefined) ? env.split(',') : default_value;
+}
+
+// Used only if https is disabled
+config.pep_port = process.env.PEP_PROXY_PORT || 5556;
+
+// Set this var to undefined if you don't want the server to listen on HTTPS
+config.https = {
+  enabled: toBoolean(process.env.PEP_PROXY_HTTPS_ENABLED, false),
+  cert_file: 'cert/cert.crt',
+  key_file: 'cert/key.key',
+  port: process.env.PEP_PROXY_HTTPS_PORT || 443,
+};
+
+//IDM configration
+config.idm = {
+  host: process.env.PEP_PROXY_IDM_HOST || '180.179.214.199',
```

Comment:
[Suggestion] why these i.p are hard coded why not configured in a configuration file then read here.

Reply:
these ip are automatically configured once the edge is started.